src/main/kotlin/org/rust/ide/intentions/JoinListIntention.kt

```diff
@@ -5,21 +5,21 @@
 
 package org.rust.ide.intentions
 
+import com.intellij.openapi.editor.Document
 import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
 import com.intellij.psi.PsiElement
-import org.rust.lang.core.psi.RsExpr
-import org.rust.lang.core.psi.RsValueArgumentList
-import org.rust.lang.core.psi.RsValueParameter
-import org.rust.lang.core.psi.RsValueParameterList
+import com.intellij.psi.PsiWhiteSpace
+import org.rust.lang.core.psi.*
 import org.rust.lang.core.psi.ext.RsElement
 import org.rust.lang.core.psi.ext.endOffset
 import org.rust.lang.core.psi.ext.startOffset
 
 abstract class JoinListIntentionBase<TList : RsElement, TElement : RsElement>(
     listClass: Class<TList>,
     elementClass: Class<TElement>,
-    intentionText: String
+    intentionText: String,
+    private val isNeedSpaceWithinParen: Boolean = false
```

Comment:
I'm not sure how clear this name is. Maybe use two parameters
```Kotlin
    private val prefix: String = "",
    private val suffix: String = ""
```
instead? Or maybe rename it somehow to make clear that it is only about leading and trailing spaces.